lib/takeaway.rb

```diff
@@ -0,0 +1,30 @@
+class Takeaway
+  attr_accessor :dishes, :order
+
+  def initialize
+    @dishes = {
+      "Beef Wellington" => 10.50,
+      "Crab Pappardelle Arrabbiata" => 8.25,
+      "Duck à L'Orange" => 9.75,
+      "Sweet & Sour Pork" => 7.00,
+      "Miso Steak" => 8.75,
+      "Beef Bourguignon" => 10.50,
+      "Salt & Pepper Squid" => 9.25,
+    }
+    @order = []
+  end
+
+  def menu
+    @dishes.each do |dish, price|
+      print "#{dish} : £#{price}\n"
```

Comment:
I really like the simplicity of this code, it is very self-explanatory and easily understandable.